constraints/miden-vm/range_checker_8bit.air

```diff
@@ -0,0 +1,32 @@
+mod RangeChecker8BitAir
+
+### Helper functions ##############################################################################
+
+# Set to 1 when we don't include the value into the running product.
+fn f0(s0: scalar, s1: scalar) -> scalar:
+    return !s0 & !s1
+
+
+# Set to 1 when we include the value into the running product.
+fn f1(s0: scalar, s1: scalar) -> scalar:
+    return s0 & !s1
+
+
+# Set to 1 when we include two copies of the value into the running product.
+fn f2(s0: scalar, s1: scalar) -> scalar:
+    return !s0 & s1
+
+
+# Set to 1 when we include four copies of the value into the running product.
+fn f3(s0: scalar, s1: scalar) -> scalar:
+    return s0 & s1
+
+
+### 8-bit Range Checker Air Constraints #################################################################
+
+ev range_checker_8bit(main: [t, s0, s1, v, p0]):
```

Comment:
`p0` should be a part of the auxiliary trace. I'd also rename this to just `range_checker`. So, this would look something like this:
```
ev range_checker_8bit(main: [t, s0, s1, v], aux: [p0]):
```

Reply:
But won't such a name interfere with the work of the evaluator of the main, already implemented range checker? Because it also has `range_checker` name.